_posts/technotes/2017-09-07-spelling-release.md

```diff
@@ -0,0 +1,88 @@
+---
+name: spelling-release
+layout: post_discourse
+title: "Spelling 1.0: quick and effective spell checking in R"
+date: 2017-09-07
+authors:
+  - name: Jeroen Ooms
+categories: blog
+tags:
+- r
+- packages
+- software
+- spelling
+- text
+---
+
+
+The new rOpenSci [spelling](https://cran.r-project.org/web/packages/spelling/index.html) package provides utilities for spell checking common document formats including latex, markdown, manual pages, and description files. It also includes tools especially for package authors to automate spell checking of R documentation and vignettes.
```

Comment:
uppercase `DESCRIPTION`?

Reply:
Hmm I don't think that will make it more readable.

Reply:
it's not about readability, it's always uppercase in a package